nmrfx-structure/src/main/java/org/nmrfx/structure/chemistry/energy/Dihedral.java

```diff
@@ -717,33 +707,6 @@ public void putInitialAngles(boolean useRandom) throws InvalidMoleculeException
         molecule.genCoords(false, null);
     }
 
-    public void printAngleTest() {
-        List<Atom> pseudoAngleAtoms = molecule.getPseudoAngleAtoms();
```

Comment:
The method was for printing and was not used anywhere, the only place I found that might have been a reference were commented out lines which had been commented out since 2017 so removed the method